server/src/test/java/org/elasticsearch/cluster/routing/allocation/RebalanceAfterActiveTests.java

```diff
@@ -131,8 +128,8 @@ public Long getShardSize(ShardRouting shardRouting) {
         // both primary and replica should not be rebalanced at once so 5 replicas should start moving
         // unless we computed the balance where one of the indices already have both primary and replica on desired nodes
         // in such case only 4 shards are immediately relocating
-        assertThat(shardsWithState(clusterState.getRoutingNodes(), STARTED).size(), oneOf(5, 6));
-        assertThat(shardsWithState(clusterState.getRoutingNodes(), RELOCATING).size(), oneOf(4, 5));
+        assertThat(shardsWithState(clusterState.getRoutingNodes(), STARTED), hasSize(2));
+        assertThat(shardsWithState(clusterState.getRoutingNodes(), RELOCATING), hasSize(8));
```

Comment:
The comment seems incorrect given this change.

Reply:
Yeap, missed it